types/validator_set_test.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 func TestValidatorSetBasic(t *testing.T) {
+	// empty or nil validator lists are allowed,
```

Comment:
Was this line is duplicated by accident?

Reply:
yup :)